int32 VI_GearInfo  
int32 VI_BrakeInfo  
int32 VI_Button1  
int32 VI_Button2  
int32 VI_HandBrakeSt  
int32 VI_JerkSt  
float64 VI_AccelPedalPosition  
float64 VI_FrontSteeringAngle  
float64 VI_RemainingTimes  
float64 VI_VehicleSpeed  
float64 SI2_LongitudinalAccel  
float64 SI2_LateralAccel  
float64 SI2_YawRate  



float64 SR_CurrentSteeringAngle 
float64 SR_CurrentSteeringSpeed 
float64 SR_HandTorque 
int32 SR_HandTorqueSign 
int32 SR_WorkMode 
float64 SR_HandTorqueLimit 
int32 SR_Error
int32 SR_Warning 
float64 SR_LiveCounter 